### DISCUSSION TITLE
Characters

### DISCUSSION THREAD

**Discussion by WalkerLukas** [B-Team-Organisation/Fantasy-Chess#106] [Ideas]
# Character Design Ideas

Put all your character design ideas here:

## For now:
One attack pattern per character with one damage value per character, subpatterns allowed.
Try to keep the stats in the limits defined by the "[Character Design Limitations.md](https://github.com/B-Team-Organisation/Fantasy-Chess/blob/dev/documentation/gameplay/Character%20Design%20Limitations.md)"

## For patterns:
p in the pattern marks the piece Position.
Every char not present in  the subpattern mappings represents a single tile attack.
Every char present in the subpattern mappings stands for an attack in the shape of the subpattern mapping.
Subpattern mappings are written like key/value pairs.
The char of the subpattern parks the center of the subpattern mapping.
Subpatterns can include other subpatterns.

### Template:

````
- **Name:**
- **Description:**
- **Hitpoints:**
- **Damage:**
- **Attack patterns:**
```
Pattern here
```
Short description
- **Subpatterns:**
```
Subpatterns here
```
Short description
- **Movement pattern:**
```
Pattern here
```
Short description
````

---

### Example:

**Name:** Limping Goblin  
**Description:** Wounded his right leg in combat, fights with his crutches  
**Hitpoints:** 15  
**Damage:** 5  
**Attack patterns:**
```
1
p
```
Can only attack in front of him

**Subpatterns:**  
```
"1":

1x
 x
```
Swings infront and right of him

**Movement pattern:**  
```
xx  
xp
```
Can only move forward and left

**Comment by WalkerLukas** [comment]
- **Name:** Chaos Goblin
- **Description:** A goblin whos only goal is chaos
- **Hitpoints:** 10
- **Damage:** 10
- **Attack patterns:**
```
 x x 
x   x
  p
x   x
 x x
```
Can attack one of many chaotic tiles
- **Subpatterns:**

- **Movement pattern:**
```
x     x
 x   x

   p

 x   x
x     x
```
Can jump diagonally

**Comment by WalkerLukas** [comment]
- **Name:** Stablin
- **Description:** Likes to stab others in their back
- **Hitpoints:** 21
- **Damage:** 25
- **Attack patterns:**
```

 p
xxx
```
Stabs others
- **Subpatterns:**
- **Movement pattern:**
```
x x x
  
  x
  p
 xxx
```
Can jump ahead far and follow slowly from behind

**Comment by WalkerLukas** [comment]
- **Name:** Goblbomber
- **Description:** Likes to experiment with explosives
- **Hitpoints:** 28
- **Damage:** 8
- **Attack patterns:**
```
 +++
++ ++
+ p +
++ ++
 +++
```
Can throw his bombs
- **Subpatterns:**
```
"+":
 x 
x+x
 x
```
Bombs explode in a plus shape
- **Movement pattern:**
```
xxx
xpx
xxx
```
Moves slowly with his heavy bombs

**Comment by WalkerLukas** [comment]
- **Name:** Bloodlust Goblin
- **Description:** A goblin that loves the fight
- **Hitpoints:** 30
- **Damage:** 15
- **Attack patterns:**
```
xxx
xpx
xxx
```
Hits on of the tiles next to him
- **Subpatterns:**
- **Movement pattern:**
```
 x
xxx
xpx
```
Never leaves the fight

**Comment by WalkerLukas** [comment]
- **Name:** Goblin Archer
- **Description:** Likes to keep his distance
- **Hitpoints:** 20
- **Damage:** 5
- **Attack patterns:**
```
xxxxx
xxxxx
 xxx
 xxx

  p
```
Attacks from far away, panics up close
- **Subpatterns:**
- **Movement pattern:**
```
x x
 p
xxx
x x
```
Rarely engages, rather runs

**Comment by Albanovukelaj17** [comment]
- **Name:** Icarus
- **Description:** An assasin that can follow you even in your dreams
- **Hitpoints:** 10
- **Damage:** 15
- **Attack patterns:**
```
xxx 
xpx  
xxx
```
can attack one of the surrounding Tiles
- **Subpatterns:**
- **Movement pattern:**
```
        x
    x   x   x
     x  x  x
      x x x
       xxx
        p
       xxx
      x x x
     x  x  x
    x   x   x
        x
```
Can move up to 5 vertical tiles, up to 4 diagonally

**Comment by Albanovukelaj17** [comment]
- **Name:** Morgana
- **Description:** A old which who who carries flames in her hand
- **Hitpoints:** 25
- **Damage:** 5
- **Attack patterns:**
```
  + 
  +
  p
```
controls the two fields before her
- **Subpatterns:**
```
"+":
xxx
x+x
```
- **Movement pattern:**
```
 x
xxx
 p
 x
```
burn the enemy near her

**Comment by Albanovukelaj17** [comment]
- **Name:** Gaius the Castle
- **Description:** Colossal Barrier 
- **Hitpoints:** 40
- **Damage:** 5
- **Attack patterns:**
```
  +
  p
```
Attacks from far away, panics up close
- **Subpatterns:**
``` "+":
  x
  x
  x
  +
```
- **Movement pattern:**
```
x x
 p
x x
```
Builds a column of defence , enhancing his allies to follow

**Comment by Albanovukelaj17** [comment]
- **Name:** Fitzooth
- **Description:** Robin Hood Fitzhooth simbolizes precision and agility . His green arrow brings a lot of pain to everyone in his radius . He stands for justice and skill, focusing the most arrogant enemy.
- **Design** The character dons a green tunic with a hood, embodying the classic archer aesthetic. A quiver of arrows rests on his back, holding a bow with an arrow nocked. His green  boots are designed for both comfort and agility.
- **Example:**![fitzooth](https://github.com/user-attachments/assets/694d71d2-6af4-42af-a435-65b600643ee8)
- **Hitpoints:** 20
- **Damage:** 17
- **Attack patterns:**
```
 +++++
  +++

+  p  +
```
Far target reacher, damages a lot 
- **Subpatterns:**
- **Movement pattern:**
```
  x
 x p x
    x 
```
Moves in a unique way to stay unpredictable

**Comment by Albanovukelaj17** [comment]
- **Name:** Prometheus 
- **Description:** Prometheus symbolizes selfless sacrifice and defiance in the pursuit of progress. He risked divine wrath to gift humanity fire.
 Despite his eternal punishment and torment by Zeus, Prometheus endured the pain for the greater good, embodying resilience and the cost of knowledge.

- **Design:** His right hand, engulfed in a vivid blue flame, reflects his power and his defiance against authority. The minimalist, clothing symbolizes simplicity and focus on his mission. His black sandals represents ancient aesthetic. He could be bald(simpler)
- **Example**
<img width="442" alt="prometheus" src="https://github.com/user-attachments/assets/79c03225-3ecc-495f-bd94-c3ec10eabf02">

- **Hitpoints:** 10
- **Damage:** 20
- **Attack patterns:**
```
  p(+)
```
Blows himself up doing area damage
- **Subpatterns:**
``` "+":
  xxx
 xx+xx
  xxx
```
- **Movement pattern:**
```
x      x
  x  x
   p
   x
   x
```
Runs into enemy lines

**Comment by dania019** [comment]
- **Name:** Dark Knight
- **Description:** moves in silence
- **Hitpoints:** 20
- **Damage:** 15
- **Attack patterns:** 
```
x-
     P
        -x
--------------
        - x  
     P
x-

```

- **Subpatterns:**
```
"+":
+ - + - +
     |
    +
    P

```
- **Movement pattern:**
```
xx      x  x
      x
      p
  xx          x
         x 

```
shoots them with sharp batman blades ... silently

**Comment by dania019** [comment]
- **Name:** Hook
- **Description:** Best Friend of Lightening McQueen
- **Hitpoints:** 17
- **Damage:** 17
- **Attack patterns:** 
     attacks by driving
```
  -
| P

```

- **Subpatterns:**
```
"-":
xxxx-xxxx
---
"|":
x
x
x
x
|
x
x
x
x
```
- **Movement pattern:**
```
x-         
     p
         -x 
```

**Comment by adbennis** [comment]
- **Name:** Flash
- **Description:** Can hit you from everywhere
- **Hitpoints:**  17
- **Damage:** 6
- **Attack patterns:**
```
   p(+)
```
activates his attack with himself in the center 
- **Subpatterns:**
 ```
"+": 
       x
     x
   x + x
     x 
   x
```
deals damage on every tile in form of a lightning
- **Movement pattern:**
```
        x
        x
        x
  x x x p x x x
        x
        x
        x
```
runs thru the field

**Comment by dania019** [comment]
- **Name:** Blossom <3
- **Description:**  attacks a lot damages little
- **Hitpoints:** 40
- **Damage:** 6
- **Attack patterns:** 
     long distance attacks from current position
```
P(+)

```

- **Subpatterns:**
```
"+":

 xx xx
 x + x
  x x
   x
```
- **Movement pattern:**
```
  x x 
  xPx  
   x  
    
```

**Comment by adbennis** [comment]
- **Name:** Santa
- **Description:** Has presents for evrybody
- **Hitpoints:** 40
- **Damage:** 10
- **Attack patterns:**
```
+                   +



          p         



+                   +  
```
better not hide in corners, santa finds you anywhere
- **Subpatterns:**
```
"+":

x  x   x
x  +   x
x  x   x
```
throws his present and hits every tile in close range
- **Movement pattern:**
```
  x
x p x
  x
```
lost his sled

**Comment by WalkerLukas** [comment]
MVP Chars:

- **Tank: Blossom**
- **Ranger: Fitzooth**
- **Assassin: Stablin**
- **Warrior: Flash**
- **AoE/Mage/Bomber:Bombo**
- **Suicide Bomber: Prometheus**
